cmd/ipfs/main.go

```diff
@@ -132,7 +132,7 @@ func main() {
 		if invoc.cmd != nil {
 			// we need a newline space.
 			fmt.Fprintf(os.Stderr, "\n")
-			printMetaHelp(os.Stderr)
+			printHelp(false, os.Stderr)
```

Comment:
`printMetaHelp` will print "Use 'ipfs %s --help' for information about this command", while `printHelp(false, writer)` will print the command's short help (subcommands, tagline, etc).